SpiderClientConverter/MainForm.cs

```diff
@@ -809,8 +812,14 @@ private void DatHex_ValueChanged(object sender, EventArgs e)
         private void SprHex_ValueChanged(object sender, EventArgs e)
         {
             SprSignature = (uint)sprHex.Value;
+        }
+
+        private void button1_Click(object sender, EventArgs e)

```

Comment:
Button and function should have a better name. Perhaps `AutoImport` for the button and `autoImport_Click` for the function.